src/benchmark_metrics.py

```diff
@@ -17,8 +17,8 @@
 # Throughput, unit: images/second
 TENSORFLOW2_SM_TRAINING_CPU_1NODE_THRESHOLD = 30
 TENSORFLOW2_SM_TRAINING_CPU_4NODE_THRESHOLD = 20
-TENSORFLOW2_SM_TRAINING_GPU_1NODE_THRESHOLD = 5000
-TENSORFLOW2_SM_TRAINING_GPU_4NODE_THRESHOLD = 4000
+TENSORFLOW2_SM_TRAINING_GPU_1NODE_THRESHOLD = 3000
```

Comment:
Is this change just for cu11 or also reflective of other cuda versions? I.e. if we move the buildspec back to cu102, will these new numbers be accurate, or will they also need to be reverted?